client/src/components/App/Carousel.jsx

```diff
@@ -0,0 +1,35 @@
+import React, { useState } from "react";
+import PropTypes from "prop-types";
+
+function Carousel({ items, size }) {
```

Comment:
Some good general style conventions, put a comment on public exports. Carousel is pretty self explanatory, but could be nice for more complicated ones, and in general for the props that are being passed

Reply:
Good call. Added some documentation. The Carousel is shared across the app, but would you suggest documentation on components that only I will be using too?